src/logger/httpLoggerRules_test.go

```diff
@@ -286,195 +286,662 @@ func TestUsesRemoveRules(t *testing.T) {
 // test uses remove if rules
 
 func TestUsesRemoveIfRules(t *testing.T) {
```

Comment:
Don't know why Git thinks I rewrote this function, I definitely did not.

Reply:
sorry, that was me

Reply:
Sorry, that was me :]